src/components/application_manager/src/application_data_impl.cc

```diff
@@ -572,6 +573,15 @@ void DynamicApplicationDataImpl::RemoveChoiceSet(uint32_t choice_set_id) {
     delete it->second;
     choice_set_map_.erase(choice_set_id);
   }
+
+  ChoiceSetAllowedMap::iterator choise_id_it =
```

Comment:
@IGapchuk you can use `auto` here

Reply:
@ZhdanovP Fixed in c03d2e5028617d584edb7e8ca84aa2fc3b1139f2.